spec/features/admin_comment_spec.rb

```diff
@@ -25,9 +25,9 @@
       end
 
       within('.admin-comments') do
-        expect(page).to have_css('form#new_admin_comment', visible: false)
+        expect(page).to have_css('form#new_admin_comment', visible: :all)
```

Comment:
Just out of curiosity, did `visible: :hidden` work here by chance?  I know `:all` is functionally equivalent to `false`, but I think what we were trying to test for was `:hidden` in these cases.

Reply:
It wasn't clear to me offhand which of these tests were looking for exclusively hidden elements, so I just went with the previous behavior 🤷‍♂️ .

Reply:
I'll push something up shortly that will hopefully be more in-line with the intent.